howto-gallery/scripts/Signs.ts

```diff
@@ -21,9 +21,9 @@ export function addSign(log: (message: string, status?: number) => void, targetL
 
   signBlock.setPermutation(signPerm);
 
-  const signComponent = signBlock.getComponent("minecraft:sign") as mc.BlockSignComponent;
+  const signComponent = signBlock.getComponent("minecraft:sign");
```

Comment:
I believe we have `BlockComponentTypes` and there are other places that could benefit from this too. Same for ItemComponentTypes and EntityComponentTypes